content/docs/capabilities/mcp.mdx

```diff
@@ -0,0 +1,727 @@
+---
+title: 'Model Context Protocol (MCP) Support'
+sidebar_label: 'Model Context Protocol (MCP)'
+lang: en-US
+description: 'Secure access to Model Context Protocol servers through Pomerium, enabling AI agents to safely interact with internal resources via standardized interfaces.'
+keywords: [MCP, model context protocol, AI agents, LLM, AI gateway]
+---
+
+# Model Context Protocol (MCP) Support
+
+Pomerium provides secure access to Model Context Protocol (MCP) servers, enabling AI agents and applications to safely interact with your internal resources through standardized interfaces. This capability allows you to expose local databases, APIs, and services to external AI clients while maintaining strict authentication and authorization controls.
+
+## Overview
+
+Model Context Protocol (MCP) is an open standard that allows AI agents to securely connect to external data sources and tools. Pomerium's MCP integration acts as a secure gateway between MCP clients (like Claude.ai, OpenAI APIs, or custom applications) and your internal MCP servers.
+
+:::warning Experimental Feature
+
+MCP support is currently an experimental feature only available in the `main` branch or Docker images built from `main`. To enable MCP functionality, you must set the following feature flag in your Pomerium configuration:
+
+```yaml
+runtime_flags:
+  mcp: true
+```
+
+:::
+
+### Key Benefits
+
+- **Secure Remote Access**: Expose internal MCP servers to external AI clients without compromising security
+- **OAuth Integration**: Seamlessly handle upstream OAuth flows for services requiring authentication
+- **Centralized Authorization**: Apply consistent access policies across all MCP endpoints
+- **Token Management**: Automatic token refresh and secure credential handling
+- **Observability**: Complete visibility into AI agent interactions with your resources
+
```

Comment:
Even though this is a Capability page, I think we need a Call to Action here pointing folks to the mcp-demo-app to try it out.

Reply:
I think we should also add to the main README a link directly to these docs that we support agentic access for people just checking out the repo.